server/data/projects.js

```diff
@@ -94,6 +103,65 @@ const createProject = async (projectObjParam, user) => {
 	return createdProject;
 };
 
+const updateProject = async (projectObjParam, id, user) => {
+	id = isValidObjectId(id);
+	let projectCheck = await getProjectById(id);
```

Comment:
you need to validate all values coming into the function before performing any of the operations